[CI] Change changelog fragment detection (#819)
mckinsey/vizro: vizro-core/docs/pages/explanation/contributing.md

* [`hatch run pypath`](#hatch-run-pypath) shows the path to the Python interpreter.
* [`hatch run example`](#hatch-run-example) runs an example dashboard on port 8050 that hot-reloads while you edit it. On GitHub Codespaces, this runs automatically on startup.
* [`hatch run lint`](#hatch-run-lint) checks and fixes code quality and formatting. This is included in CI checks.

> * [`hatch run changelog:add`](#hatch-run-changelogadd) generates a new changelog fragment. This is included in CI checks and required for all changes to source code.

Reviewer: ```suggestion
* [`hatch run changelog:add`](#hatch-run-changelogadd) generates a new changelog fragment. Changelog inclusion is checked by CI and required for all changes to source code.
```

### `hatch run changelog:add`

> `hatch run changelog:add` generates a new changelog fragment. This is included in CI checks and required for all changes to source code.

Reviewer: ```suggestion
`hatch run changelog:add` generates a new changelog fragment. Changelog inclusion is checked by CI and required for all changes to source code.
```
